x-pack/plugins/apm/ftr_e2e/ftr_config.ts

```diff
@@ -6,8 +6,11 @@
  */
 
 import { FtrConfigProviderContext } from '@kbn/test';
-
 import { CA_CERT_PATH } from '@kbn/dev-utils';
+
+// Used to spin up a docker container with package registry service that will be used by fleet
+export const packageRegistryPort = 1234;
```

Comment:
Will this cause the e2e's to run slower?

Reply:
I actually don't know how FTR is going to handle that if it spins up a docker container async with elasticsearch and kibana.